tests/Collection/CollectionFunctionalTest.php

```diff
@@ -32,28 +33,28 @@ class CollectionFunctionalTest extends FunctionalTestCase
     /**
      * @dataProvider provideInvalidDatabaseAndCollectionNames
      */
-    public function testConstructorDatabaseNameArgument($databaseName): void
+    public function testConstructorDatabaseNameArgument($databaseName, string $expectedExceptionClass): void
     {
-        $this->expectException(InvalidArgumentException::class);
+        $this->expectException($expectedExceptionClass);
         // TODO: Move to unit test once ManagerInterface can be mocked (PHPC-378)
         new Collection($this->manager, $databaseName, $this->getCollectionName());
     }
 
     /**
      * @dataProvider provideInvalidDatabaseAndCollectionNames
      */
-    public function testConstructorCollectionNameArgument($collectionName): void
+    public function testConstructorCollectionNameArgument($collectionName, string $expectedExceptionClass): void
```

Comment:
I just realized that the data provider here should be named `provideInvalidCollectionNames` instead. I don't recall if an older version of this test took a database name, but it certainly doesn't now.

Reply:
`provideInvalidDatabaseAndCollectionNames` is a single data provider for both database and collection names that is used in both `testConstructorDatabaseNameArgument` and `testConstructorCollectionNameArgument` - I suppose that's why the data provider name includes both "database" and "collection".

Reply:
Oh, you're right. I misunderstood how these were being used and thought they were appearing in various Operation tests instead of just the Database and Collection functional tests. All good here.